docs/sources/developer/architecture/components/api.md

```diff
@@ -0,0 +1,78 @@
+# API
+
+The Faro API is the primary interface for end-users to interact with Faro. It is the central collector of signals, and
+also exposes helper methods for extensions such as metas.
+
+## Signals
+
+### Events
+
+Events are signals that can be used to record user interactions, such as mouse clicks, page navigation, or client-side
+application events like page transitions.
+
+Methods:
+
+- `pushEvent` - sends an event
+
+### Exceptions
+
+Exceptions are signals that are represented by errors which occurred in an app.
+
+When an exception is sent, the API will pass it through a stacktrace parser to extract the relevant information from it.
```

Comment:
This sentence is a bit confusing imo

```suggestion
When an exception is injected by the API, its minified stacktrace will be transformed to the original source, in order to extract relevant information from it.
```

Reply:
Nope.

The exception is not injected, but it is "sent" using `sendError` method.
The stacktrace parser does not receive a minified stacktrace but the original source. It is actually parsing the original source to extract the filename, the function name, the col number and line number.